src/frontend/state/ContextProvider.tsx

```diff
@@ -101,7 +101,11 @@ const initialContext: ContextType = {
   disableDialogBackdropClose: false,
   setDisableDialogBackdropClose: () => null,
   disableAnimations: false,
-  setDisableAnimations: () => null
+  setDisableAnimations: () => null,
+  storeConfigs: {
+    storeConfigs: [],
+    runnerToDisplayName: () => 'Other'
+  }
```

Comment:
I'd recommend against adding new things to the global state, instead either using individual Zustand stores or adding them to the `GlobalStateV2` store